.coveragerc

```diff
@@ -581,6 +581,7 @@ omit =
     homeassistant/components/niko_home_control/light.py
     homeassistant/components/nilu/air_quality.py
     homeassistant/components/nissan_leaf/*
+    homeassistant/components/niu/*
```

Comment:
Tests are required for the config flow. Our scaffold script can help with some initial config flow tests.

Reply:
Do you have a link for the script? Didn't find anything useful in the docs.

Reply:
```bash
python3 -m script.scaffold config_flow --integration niu
```
The test will be generated in `tests/components/niu/test_config_flow.py`.
You can delete the `EXAMPLE_*` files.